src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -323,11 +351,54 @@ describe("FunctionsEmulatorRuntime", () => {
 
         await runtime.exit;
       }).timeout(TIMEOUT_MED);
+
+      // it("should merge .initializeApp arguments from user", async () => {
```

Comment:
Travis guarantees this environment:
```
CI=true
TRAVIS=true
CONTINUOUS_INTEGRATION=true
```

Either deleted the commented out code or prepend a travis check.